### DISCUSSION TITLE
Leave Request restriction

### DISCUSSION THREAD

**Discussion by asifrana1972** [horilla-opensource/horilla#210] [Q&A]
If we assign leave request **create** rights to someone, he can view all employee and apply leaves to anyone. There must be department level restriction. He/she can view and apply for those department level employees.

**Comment by horilla-opensource** [comment]
> If we assign leave request **create** rights to someone, he can view all employee and apply leaves to anyone. There must be department level restriction. He/she can view and apply for those department level employees.

Hi @asifrana1972 ,
The feature is now updated in the latest code base. 
Please check and let us know.

With Regards,
**Team Horilla**
